torch/_refs/__init__.py

```diff
@@ -3298,11 +3299,11 @@ def native_layer_norm(
         + str(input.shape),
     )
 
-    input = input.contiguous()
+    input = contiguous(input)
```

Comment:
if we do the c++ change we do not need those right?

note that this is not just meta tracing. 
@bdhirsh  how bad would it be calling python instead of c++ here, i guess not that bad since we already went to python ?